tests/roles/backend_services/templates/openstack_control_plane.j2

```diff
@@ -57,8 +57,10 @@ spec:
       ironicConductors: []
 
   keystone:
-    enabled: false
-    template: {}
+    template:
```

Comment:
Restore this file to have `enabled: false`.
Keystone should be patched during the adoption phase, which means you have to apply that config here [1]

[1] https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/roles/keystone_adoption/defaults/main.yaml